ferdinand-porsche-fern-fachhochschule-german.csl

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-AT" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Ferdinand Porsche Fern-Fachhochschule (German)</title>
+    <title-short>Ferdinand Porsche FernFH (German)</title-short>
+    <id>http://www.zotero.org/styles/ferdinand-porsche-fern-fachhochschule-german</id>
+    <link href="http://www.zotero.org/styles/ferdinand-porsche-fern-fachhochschule-german" rel="self"/>
+    <link href="http://libweb.anglia.ac.uk/referencing/harvard.htm" rel="documentation"/>
```

Comment:
is there any documentation for this style available? Preferably online, but we'd take a link to a book, too, absent other options.